packages/Chem/src/analysis/chem-similarity-viewer.ts

```diff
@@ -165,16 +171,10 @@ export class ChemSimilarityViewer extends ChemSearchBaseViewer {
     }
   }
 
-  isEmptyOrMalformedValue(): boolean {
-    const malformed = !getMolSafe(this.targetMolecule, {}, getRdKitModule()).mol;
-    const empty = !this.targetMolecule || DG.chem.Sketcher.isEmptyMolfile(this.targetMolecule);
-    const moleculeError = malformed ? `Malformed` : empty ? `Empty` : '';
-    if (moleculeError) {
-      grok.shell.error(`${moleculeError} molecule cannot be used for similarity search`);
-      this.clearResults();
-      return true;
-    }
-    return false;
+  closeWithError(error: string, progressBar: DG.TaskBarProgressIndicator) {
+    grok.shell.error(`${error} molecule cannot be used for similarity search`);
```

Comment:
Basically, we can see in the grid that molecule is malformed (grey cross is shown instead of a structure). This error is like additional notification here. Should I remove it?